test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1539,13 +1684,16 @@ def load_config(self, cfg, mom_checks=True):
                 r2 = self.du.run_cmd(cmd=['cat', mom_conf], sudo=True)
                 if r1['out'] != r2['out']:
                     self.logger.info('server & mom pbs_cgroups.CF differ')
-                    just_before_import = time.time() - 1
+                    time.sleep(2)
+                    just_before_import = int(time.time())
+                    time.sleep(2)
```

Comment:
How about calling self.log_lines() with n=1 on the mom where you want to call log_match() later on, this will give you a log line from that host's mom logs. you can then use it to get a timestamp from that machine. Something like this (untested):
lastlog = self.moms_list[0].log_lines(n=1)
timestamp = float(self.logutils.convert_date_time(lastlog[0].split(";", 1)[0]))

Reply:
That is a good idea -- feel free to submit your own PR to make it all more elegant, but I'm now just going to ensure it works -- including the tests that I did not write.

As you said: your idea has merit, but it's "untested". This isn't, it's tested _ad nauseam_ by now.

Reply:
I chose not to engage in something like that because it would gain us 2 seconds in tests at the cost of readability.